extensions-contrib/materialized-view-maintenance/src/main/java/org/apache/druid/indexing/materializedview/MaterializedViewSupervisor.java

```diff
@@ -335,7 +335,8 @@ void checkSegmentsAndSubmitTasks()
     Map<Interval, String> toDropInterval = new HashMap<>(difference.entriesOnlyOnRight());
     // if some intervals are in running tasks and the versions are the same, remove it from toBuildInterval
     // if some intervals are in running tasks, but the versions are different, stop the task. 
-    for (Interval interval : runningVersion.keySet()) {
+    for (Map.Entry<Interval, String> entry : runningVersion.entrySet()) {
```

Comment:
What's the purpose of this change?

Reply:
I assumed we need to replace all the occurrences of keyset() with entrySet()?

Reply:
@hpandeycodeit see the description of the inspection (bold is mine):

> Reports iteration over the keySet() of a java.util.Map instance, **where the iterated keys are used to retrieve the values from the map**. Such iteration may be more efficiently replaced by iteration over the entrySet() of the map.

When values are not used in the loop, it doesn't make any sense to replace `keySet()` with `entrySet()`.

Reply:
reverted this and other similar change on this file